src/client/containers/ConfigurationCreateContainer/hooks/useConfigurationCreate.ts

```diff
@@ -61,13 +61,14 @@ export const useConfigurationCreate = ({
 
     useEffect(() => {
         if (isInitialized) {
-            microsoftTeams.getContext(({
-                channelId,
-                channelName,
-                tid: tenantId
-            }) => {
-                microsoftTeams.settings.getSettings(settings => {
-                    microsoftTeams.settings.registerOnSaveHandler(async saveEvent => {
+            app.getContext().then(({
```

Comment:
Nothing important but, should we also update to async, await style to improve readability ?